tests/test_model.py

```diff
@@ -14,7 +14,7 @@ def predict(self, input: str, **kwargs: dict) -> str:
             return input + " lol"
 
         @pipeline_function
-        def load(self, path: str) -> None:
+        def load(self, path: str = None) -> None:
```

Comment:
From what I can tell `load` is never called with arguments, so you could also just remove `path` here right?

```suggestion
        def load(self):
```